### PR TITLE
fix(widget): add empty value for poller filter

### PR DESCRIPTION
**Pull request by s-duret**
## Description

Fix the poller filter in the widget preferences.
We have to let the user select an empty line.

**Fixes** MON-14174

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [x] 21.04.x
- [x] 21.10.x
- [x] 22.04.x
- [x] 22.10.x (master)

### CODE REVIEW

File: www/class/centreonWidget/Params/Connector/Poller.class.php [hunk 1]
```diff
@@ -89,6 +89,7 @@ public function getListValues($paramId)
                     );
                 }
 
+                $tab = array(null => null);
```
Comment (jeremyjaouen) [inline www/class/centreonWidget/Params/Connector/Poller.class.php]:
```suggestion
                $tab = [];
```
Could you check with an empty array if this correct the bug ?

**Review by jeremyjaouen — DISMISSED** [review]

**Review by wtermellil — DISMISSED** [review]

**Review by dmyios — DISMISSED** [review]

File: www/class/centreonWidget/Params/Connector/Poller.class.php [hunk 2]
```diff
@@ -89,6 +89,7 @@ public function getListValues($paramId)
                     );
                 }
 
+                $tab = [];
```
Comment (tanguyvda) [inline www/class/centreonWidget/Params/Connector/Poller.class.php]: doesn't work on 21.10 at least 

```suggestion
                $tab = [null => null];
```

**Comment by TamazC** [comment]
The issue is already fix within https://github.com/centreon/centreon/commit/1c7dbfa07170a28d94446ca2547371826ba2ab16

@s-duret could you please check the fix and close it if OK ?

Thanks

**Comment by s-duret** [comment]
> The issue is already fix within [1c7dbfa](https://github.com/centreon/centreon/commit/1c7dbfa07170a28d94446ca2547371826ba2ab16)
> 
> @s-duret could you please check the fix and close it if OK ?
> 
> Thanks

Hello @TamazC 
Yes you're right, it the same fix.
I don't understand, my PR was done in june and the fix you gave me was done 2 weeks ago.
No matter, I will close this issue.

**Pull request closed without merging**